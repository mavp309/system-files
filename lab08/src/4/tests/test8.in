-9.01 -5.67 2.3 6.4
