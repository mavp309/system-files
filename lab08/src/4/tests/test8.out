-5.8100 -4.5200
